app/controllers/api/restful_controller.rb

```diff
@@ -17,6 +17,10 @@ def destroy_action
 
   private
 
+  def current_user
+    super || LoggedOutUser.new
```

Comment:
I like the approach.. but how does the super call work in this case?

Where do we return the real current user when they're logged in?

(I think I'm finding that I don't ever like the use of super.. but that's totally an aside and I'm happy for you to use it once I understand this)